recipe/meta.yaml

```diff
@@ -1,25 +1,25 @@
-{% set version = "1.5.0" %}
+{% set version = "1.5.1.dev0+" + GIT_FULL_HASH[0:7] %}
 
 package:
   name: pynio
   version: {{ version }}
 
 source:
-  url: https://github.com/NCAR/pynio/archive/{{ version }}.tar.gz
-  sha256: e2bae6469541662835a7d13aa0185b73b13df56c2de6ce6f8e74cb17ba9a6143
+  git_url: https://github.com/NCAR/pynio.git
+  git_rev: develop
```

Comment:
We should use the commit hash here so we can reproduce the build.

Reply:
Good call, I just updated it to use the commit hash.

Sorry for not using the other pull request, I didn't notice you had made one until after I had already created this one.

Reply:
> Sorry for not using the other pull request, I didn't notice you had made one until after I had already created this one.

No problem. I only sent that as an example. Thanks for moving this forward!